gulpfile.js

```diff
@@ -36,179 +36,190 @@ var enabledTemplates = [];
 //
```

Comment:
Do we "require" all the *.js files automatically? For the pattern template we just need the components / directives, but the build requires all the *.js files (controllers, services)